pkg/api-server/server.go

```diff
@@ -164,7 +165,11 @@ func (a *ApiServer) Start(stop <-chan struct{}) error {
 
 func SetupServer(rt runtime.Runtime) error {
 	cfg := rt.Config()
-	apiServer, err := NewApiServer(rt.ResourceManager(), definitions.All, rt.Config().ApiServer, &cfg)
+	var apiDefinitions = definitions.All
+	if cfg.Mode == config_core.Local {
+		apiDefinitions = definitions.DataPlaneDefinitions
+	}
```

Comment:
I think for local it makes sense to expose other APIs that Dataplane but in read-only mode. We want to have ability to debug if resources are synchronized.

@nickolaev WDYT?

Reply:
@tharun208 read only is for other resources than dataplane.